### ISSUE TITLE
No Jupyterlab 2.x support

### ISSUE THREAD

**Issue by thomaspernet** [aws-samples/sagemaker-run-notebook#1]
Dear, 

thanks for all your instructions and for the python module! It's very helpful and convenient.

I followed each step in the quickstart. Everything works except the Jupyter lab extension on a local machine. 

The extension  `sagemaker_run_notebook` is outdated, and no possibility to update it. 

```
(base) thomas@Thomass-MacBook-Pro Deployment % jupyter labextension list                                                              
JupyterLab v2.1.2
Known labextensions:
   app dir: /Users/thomas/anaconda3/share/jupyter/lab
        @aquirdturtle/collapsible_headings v2.2.0  enabled  OK
        @jupyterlab/git v0.20.0  enabled  OK
        @jupyterlab/toc v4.0.0  enabled  OK
        @ryantam626/jupyterlab_code_formatter v1.3.1  enabled  OK
        jupyterlab-jupytext v1.2.1  enabled  OK
        jupyterlab-plotly v4.8.1  enabled  OK
        nbdime-jupyterlab v2.0.0  enabled  OK
        sagemaker_run_notebook v0.13.0  enabled   X  

   The following extension are outdated:
        sagemaker_run_notebook
        
   Consider running "jupyter labextension update --all" to check for updates.

(base) thomas@Thomass-MacBook-Pro Deployment % jupyter labextension update --all
Failed to fetch package metadata for 'sagemaker_run_notebook': <HTTPError 404: 'Not Found'>
No compatible version found for sagemaker_run_notebook!
```

**Comment by tomfaulhaber** [comment]
Oh, this error is because you're running JupyterLab 2.x and the plugin only supports 1.x. We're currently figuring out whether we're going to support 2.x or just skip forward to 3.x, which is coming soon and will hopefully end the run of version incompatibilities.

This should have been called out in the Quick Start and I have now added it (see commit 99b1b03).

If you wish to try out the extension, you can revert to the latest version of JupyterLab 1.x with the following commands:

```sh
$ pip uninstall jupyterlab
$ pip install 'jupyterlab<2'
```

You'll probably need to reinstall the other extensions as well after this to get to the right versions.

**Comment by thomaspernet** [comment]
Okay, thank you for your reply. 

I'll stay with the command line, and wait for you to update the extension with jupyterlab 3.+

**Comment by tomfaulhaber** [comment]
Let's leave this issue open since we do need to resolve it either by supporting JupyterLab 2 or JupyterLab 3.

**Comment by naterush** [comment]
Any updates on the plan to support JupyterLab 2 or 3? I'm a JupyterLab extension developer, and we've got SageMaker customers who want a deployment - but we only currently support JupyterLab 2. Any advice would be greatly appreciated!

**Comment by jbeyer16** [comment]
Ditto on this. Would love to see JupyterLab 2 or 3 supported soon!

**Comment by naterush** [comment]
Wanted to follow up on this. Any planned timeline for support? Customers are knockin' down the door!

**Comment by Techie-Fox** [comment]
Personally I am struggling to see the strings in Dark Theme on JupyterLab 1.x due to colour deficiency and inhibited red perception. It was fixed in 2.x but we're stuck on 1.x so either I look at a bright screen all day or squint at nearly-invisible text!

**Comment by mnicstruwig** [comment]
Adding to the growing chorus. We're running into major issues with clients who want to use widgets and libraries that do no support `jupyterlab < 3.0`

**Comment by jeremyallenjacobson** [comment]
I would like to use NV Dashboard, a JupyterLab extension for displaying GPU usage dashboards. It requires > Jupyter lab version 3.0. Is Sagemaker going to make this upgrade soon?

**Comment by S-UP** [comment]
Same here. We depend on some JupyterLab extensions supported only on version 3. We are considering multi-cloud solutions, e.g. Google Cloud Platform, to have this capability.

**Comment by tomfaulhaber** [comment]
JupyterLab 2 support is added in commit #4349735b909d3af2520e4e0c33fe670fc92d3332 and released in [v0.20.0](https://github.com/aws-samples/sagemaker-run-notebook/releases/tag/v0.20.0). It turned out to make sense to move the code from JupyterLab 1.x to JupyterLab 2.x before tackling JupyterLab 3.x, so I figured we should release it so that anyone using JupyterLab 2.x would have a release to consume.

I've opened Issue #36 to address moving to JupyterLab 3.
